CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0 (2017-10-10)
+
+- Added a method to expose the `toggleOpen` method from the widget.
+
```

Comment:
Add a reference to the related PR, please? (I know we're not doing it below; that's a mistake.)

Reply:
👍